reference/forms/types/collection.rst

```diff
@@ -53,13 +54,9 @@ in a form. For example, suppose you have an ``emails`` field that corresponds
 to an array of email addresses. In the form, you want to expose each email
 address as its own input text box::
 
-    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
-    use Symfony\Component\Form\Extension\Core\Type\EmailType;
-    // ...
-
-    $builder->add('emails', CollectionType::class, array(
+    $builder->add('emails', 'Symfony\Component\Form\Extension\Core\Type\CollectionType', array(
```

Comment:
@HeahDude It looks like I was mistaken here and we at some point agreed on using the `class` constant even in the 2.8 docs (see http://symfony.com/doc/2.8/book/forms.html, for example). Sorry for the confusion.